cmd/root.go

```diff
@@ -64,7 +63,7 @@ var rootCmd = &cobra.Command{
 	Short:   rootCmdShortDescription,
 	Long:    rootCmdLongDescription,
 	PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
-
+		ste.UploadTryTimeout, _ = time.ParseDuration(glcm.GetEnvironmentVariable(common.EEnvironmentVariable.UploadTryTimeout()) + "m")
```

Comment:
So if the environment variable is empty or not set incorrectly, what happens?

We need a default value.

Reply:
There is a default value for UploadTryTimeout in xfer.go (line 39) but I added error validation for wrong or empty environment variable.